praktikumsplaner-frontend/frontend/src/components/assign/PraktikumsstelleCardZuweisung.vue

```diff
@@ -255,4 +258,9 @@ function getNwkColor(nwk: Nwk): string {
     bottom: 10px;
     right: 10px;
 }
+.chip-text {
+    min-width: 30vmin;
+    max-width: 45vmin;
+    width: 90%;
```

Comment:
the width should not be necessary. If you drop it, then the name should use all the space up to the close button.